package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "brainly-style-guide",
-  "version": "218.0.1",
+  "version": "218.0.1-beta-ads-dialog.1",
```

Comment:
Marking to not forget removing that

Reply:
Thanks. I updated it right now to not forget :)